systers_portal/templates/community/snippets/search-snippet.html

```diff
@@ -1,3 +1,26 @@
+<script>
```

Comment:
Same here

Reply:
@sammy1997 @SanketDG @sakshi1499 I won't be able to add the javasript to separate files, as the javascript code uses template tags.

Reply:
@ritwickraj78 I see. Maybe we could add a block template here just for the javascript? Thoughts?

Reply:
Done